### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by johnsko89**

### CODE REVIEW

File: app/views/buddies/show.html.erb [hunk 1]
```diff
@@ -1,71 +1,93 @@
...
-        <% end %>
-      <% end %>
-    <% else %>
-      <p>Be the first to review this buddy!</p>
-    <% end %>
-    <% if signed_in? %>
-      <%= form_for([@buddy, @buddy.reviews.build]) do |f| %>
+<!-- Make New Review -->
+<% if signed_in? %>
+  <div class="">
```
Comment (vikram7) [inline app/views/buddies/show.html.erb]: Is this supposed to be a row class?

File: app/views/buddies/show.html.erb [hunk 2]
```diff
@@ -1,71 +1,93 @@
...
+              <% end %>
+            </div>
+          </div>
+        </div>
+      <% end %>
+    <% end %>
+    <!-- END PRINTING REVIEWS -->
+  <% else %>
+    <p>Be the first to review this buddy!</p>
+  <% end %>
```
Comment (vikram7) [inline app/views/buddies/show.html.erb]: It's great that y'all added comments to break up each of the features, but this code is nested and hard to follow because of just how much is going on. This is a really nice use case for partials instead, which is worth implementing here instead: http://guides.rubyonrails.org/layouts_and_rendering.html#using-partials

Comment (cmkoller) [inline app/views/buddies/show.html.erb]: Good point!  Is it ok if we merge as is and then make those changes in the pagination branch?
